code/__DEFINES/font_awesome_icons.dm

```diff
@@ -645,7 +645,7 @@
 #define FA_ICON_DIRECTIONS "fa-directions"
 #define FA_ICON_DISCORD "fa-discord"
 #define FA_ICON_DISCOURSE "fa-discourse"
-#define FA_ICON_DISEASE "fa-disease"
+#define FA_ICON_ERADICATE "fa-person-dots-from-line"
```

Comment:
You can't just change this file. There's an unit test that checks against it. Also, did you ensure if we have the icon? Last time I checked, we were still using the older font awesome version (I wonder why I hadn't updated it back then).

You can still assign `"person-dots-from-line"` to the `icon` var in the quirk definition anyway, the `fa-` prefix is a bit redundant since the js/inferno icon class we use does just as well without.